src/modules/qt/filter_typewriter.yml

```diff
@@ -0,0 +1,55 @@
+schema_version: 0.1
+type: filter # consumer, filter, producer, or transition
+identifier: typewriter
+title: TypeWriter
+version: 0.3.4
+copyright: Copyright (C) Rafal Lalik
+license: GPL
+language: en
+url: none
+creator: Rafal Lalik <rafallalik@gmail.com>
+tags:
+  - Video
+description: |
+    Typewriter effect applied to kdenlivetitler producer
+
+notes: >
+    Original development: https://github.com/rlalik/mlt_extra_modules
+
+parameters:
+  - identifier: step_length
+    title: Distance between basic steps
+    type: integer
+    description: Defines how many frames it takes to display next character
+    readonly: no
+    mutable: yes
+    default: 25
+
+  - identifier: sigma_step
```

Comment:
It is not `sigma_step`; it is `step_sigma`.

Reply:
Corrected.